doc/docs/concepts.md

```diff
@@ -212,3 +218,25 @@ restriction (always allow access), and then as a post-processing step adds the
 new Patient id to the client's patient access list. You can see this implemented
 in
 [`AccessGrantedAndUpdateList`](https://github.com/google/fhir-access-proxy/blob/main/plugins/src/main/java/com/google/fhir/gateway/plugin/AccessGrantedAndUpdateList.java).
+
+## AuditEvent logging
+
+Given that every access to the FHIR server goes through the Gateway, it's the
+ideal place to track access for auditing. The Gateway simplifies enabling
+AuditEvent logging with these key features:
+
+- Targeting the HL7 FHIR R4 specification -
+  [https://hl7.org/fhir/R4/auditevent.html](https://hl7.org/fhir/R4/auditevent.html).
+- Adherence to Basic Audit Logging Profiles (BALP) IG minimal audit patterns -
+  [https://profiles.ihe.net/ITI/BALP/index.html](https://profiles.ihe.net/ITI/BALP/index.html).
+- Storing AuditEvents in the same server as the rest of the data.
+- Generating an AuditEvent for every request processed by the Gateway, based on
+  request and response data.
+- Configurable `AuditEvent.agent[user].who` through overriding the default
+  `getUserWho` implementation in the `AccessDecision` interface.
```

Comment:
Please add a link to [the default implementation](https://github.com/google/fhir-gateway/blob/1eaacd770dd5a098588eb302dd9ef3235ce88b72/server/src/main/java/com/google/fhir/gateway/interfaces/AccessDecision.java#L81).